envs/environment.yaml

```diff
@@ -29,32 +30,14 @@ dependencies:
   - powerplantmatching>=0.4.8
   - numpy<=1.19.0 # otherwise macos fails
 
-  # Second order dependencies which should really be deps of atlite
-  - xarray
-  - netcdf4
-  - bottleneck
-  - toolz
-  - dask
-  - progressbar2
-  - pyyaml>=5.1.0
 
   # Keep in conda environment when calling ipython
   - ipython
 
   # GIS dependencies:
   - cartopy
-  - fiona
-  - proj
-  - pyshp
-  - geopandas
-  - rasterio
-  - shapely
-  - libgdal<=3.0.4
   - descartes
 
   - pip:
     - vresutils==0.3.1
     - tsam>=1.1.0
-    - git+https://github.com/PyPSA/glaes.git#egg=glaes
-    - git+https://github.com/PyPSA/geokit.git#egg=geokit
-    - cdsapi
```

Comment:
we could keep `cdsapi`

Reply:
ah, or is that an `atlite` dependency?

Reply:
only `atlite` AFAIK and enforced in the new version.

Reply:
yes!